modules/tensor_mechanics/doc/content/source/materials/ComputeMultipleInelasticStress.md

```diff
@@ -284,18 +284,27 @@ non-yielding materials (e.g. volumetric swelling) and Full consistent tangent
 operators should be implemented for yielding material models (e.g. plasticity).
 
 ### Include Damage Model
-Optionally, the effect of damage on the stress calculation can be included in the model. Another material that defines the evolution of damage should be coupled using parameter `damage_model`. Here, first the inelastic strains and corresponding effective stresses are calculated based on the undamaged properties. Afterwards, the damage index is applied on the effective stress to calculate the damaged stress. This captures the effect of damage in a material undergoing creep or plastic deformation.
+
```

Comment:
@bwspenc this empty line between the header and the paragraph was the main type of fix I made; if there is no empty line between the header line and the paragraph, the entire paragraph renders as header. I just fix the text wrapping with atom since I was at the paragraph anyway